### ISSUE TITLE
vehSDKTruck not set

### ISSUE THREAD

**Issue by wurzel0701** [official-antistasi-community/A3-Antistasi#669]
*Version:* 2.2.1

*Mods:* None

*Environment*: SP

*.rpt attatched?* No

*have you edited the missionfile?*: Obviously

***Issue:***

``` 7:30:07 Error in expression <esourcesFIA",5000,true];
vehInGarage = [vehSDKTruck,vehSDKTruck,SDKMortar,SDKMGS>
 7:30:07   Error position: <vehSDKTruck,vehSDKTruck,SDKMortar,SDKMGS>
 7:30:07   Error Undefined variable in expression: vehsdktruck
 7:30:07 File functions\Dialogs\fn_createDialog_setParams.sqf [A3A_fnc_createDialog_setParams]..., line 20```

Not sure how this happened at all. Started a SP Altis no mod game for testing other stuff. 
I checked the templates and it is there every time. However, this appears in the rpt before 

``` 7:30:10 "0: [Antistasi] | INFO | initServer.sqf | Proceeding to initServer"
 7:30:10 "0: [Antistasi] | INFO | initServer.sqf | Dedicated server detected"
 7:30:10 "0: [Antistasi] | INFO | initServer.sqf | Server init started"```

 the init start, so that may be it.

**Comment by SonorousMeerkat** [comment]
Ordering issue? Is that an artifact left in a file that now gets called before templates are loaded?

**Comment by Bob-Murphy** [comment]
@Spoffy Could you please check the order?
Thank you in advance.

**Comment by jaj22** [comment]
As far as I can tell, this should only happen if you somehow manage to click through the startup dialogs before initserver gets around to loading the templates. I'm not currently sure how that's possible.

**Comment by jaj22** [comment]
Ok, it's a plain ordering issue. The startup dialog code runs strictly before initserver. Immediately on close, the startup dialog code attempts to set parameters which are overwritten by initvar, or in this case using vehicle template data that's not yet available.

Straightforward solution would be to remove the difficulty-dependent code from createDialog_setParams. Could be moved to initServer/initVar or could be dropped entirely. minWeaps should probably be hard-set to 15 for SP rather than making it dependent on skillMult.

**Comment by jaj22** [comment]
Should have been closed with #712
